experimental/Schemes/CoherentSheaves.jl

```diff
@@ -1193,10 +1546,90 @@ end
   return matrix(map(presentation(M), 1))
 end
 
-function fill_with_lazy_glueings!(C::Covering, X::AbsCoveredScheme)
-  # TODO: Fill in all the implicit and restricted glueings from X.
-  # We assume that all patches in C are in a tree structure eventually 
-  # leading to affine charts of X.
-  return C
+########################################################################
+# Projectivization of vector bundles                                   #
+#
+# To any vector bundle E → X one can associate its projectivization 
+# ℙ(E) → X. In general, E will be given as a locally free 
+# AbsCoherentSheaf.
+########################################################################
+
+function projectivization(E::AbsCoherentSheaf; 
+    var_names::Vector{String}=Vector{String}(),
+    check::Bool=true
+  )
+  X = scheme(E)
+  check && (is_locally_free(E) || error("coherent sheaf must be locally free"))
+  C = trivializing_covering(E)
+  algebras = IdDict{AbsSpec, Union{MPolyQuo, MPolyRing}}()
+  on_patches = IdDict{AbsSpec, AbsProjectiveScheme}()
+  for U in patches(C)
+    F = E(U)
+    F isa FreeMod || error("modules must locally be free")
+    r = rank(F)
+    if length(var_names) == 0
+      var_names = ["s$i" for i in 0:r-1]
+    end
+    length(var_names) == r || error("number of names for the variables does not coincide with the local rank of the module")
```

Comment:
If the scheme is not connected, a locally free sheaf could have different ranks in different components? (or is that excluded?) (probably should be)

Reply:
As I see this function, it assumes that on each patch U the rank is constant, which should be reasonable even in the case of a non-connected scheme, because it has already computed a trivializing covering. 

The key question is: How do is_locally_free and trivializing_covering behave in the case of a non-connected scheme? A test for this should be added (no matter whether it is forbidden or handled correctly).